scripts/download_loco.sh

```diff
@@ -9,14 +9,14 @@ function download_url {
 }
 
 function destination_path {
-  echo "SayTheirNames/Resources/$1.lproj/Localizable.strings"
+  echo "../SayTheirNames/Resources/$1.lproj/Localizable.strings"
```

Comment:
do we need this change to the script?

Reply:
I think so. Since the script isn't in the same directory as the SayTheirNames, I kept getting the error that it couldn't find the file [language].lproj/Localizable.strings, but this managed to point it to the right place

Reply:
Ah I see. The script is to be run from the root directory of the repo, like this: `sh scripts/download_loco.sh`. Like that you shouldn't be getting any errors. 
I will revert this change here, and of you still get errors then let's fix it separately 🙂

Reply:
```suggestion
  echo "SayTheirNames/Resources/$1.lproj/Localizable.strings"
```